mephisto/client/review/review_server.py

```diff
@@ -102,39 +103,30 @@ def consume_all_data(page, limit=RESULTS_PER_PAGE_DEFAULT):
         data_point_list = []
 
         if database_task_name is not None:
-            # if reading from MephistoDB
-            data_source = mephistoDBReader()
-
-            if paginated:
-                for x in range(first_index):
-                    data = next(data_source, None)
-                    if data is None:
-                        return []
-                for x in range(limit):
-                    data = next(data_source, None)
-                    if data is None:
-                        break
-                    data_point_list.append(data)
-            else:
-                for row in data_source:
-                    data_point_list.append(row)
+            # If differnce in time since the last update to the data list is over 5 minutes, update list again
+            # This can only be done for usage with mephistoDB as standard input is exhausted when originally creating the list
+            global datalist_update_time
+            now = datetime.now()
+            if (now - datalist_update_time).total_seconds() / 60.0 > 5:
```

Comment:
This (5) should be moved to a constant near the top of the file for easy editing. Generally it's good to avoid magic numbers (could also move both by making the constant based on a second timeout).

Reply:
It would also be good to have a boolean constant flag to toggle this functionality on/off altogether

Reply:
Added both for my next PR. My bad I didn't have this in the current version.